plugwise/smile.py

```diff
@@ -47,6 +50,44 @@
 class SmileData(SmileHelper):
     """The Plugwise Smile main class."""
 
+    def update_for_cooling(self, devices: dict[str, DeviceData]) -> None:
+        """Helper-function for adding/updating various cooling-related values)."""
+        for _, device in devices.items():
+            # For Anna + cooling, modify cooling_state based on provided info by Plugwise
+            if self.smile_name == "Anna":
+                if device["dev_class"] == "heater_central" and self._cooling_present:
+                    device["binary_sensors"]["cooling_state"] = False
+                    if self._anna_cooling_active:
+                        device["binary_sensors"]["cooling_state"] = True
+
+                # Add setpoint_low and setpoint_high when cooling is enabled
+                if device["dev_class"] not in ZONE_THERMOSTATS:
+                    continue
+
+                if self.anna_cooling_enabled:
+                    sensors = device["sensors"]
+                    if self._sched_setpoints is None:
+                        sensors["setpoint_low"] = sensors["setpoint"]
+                        sensors["setpoint_high"] = float(40)
```

Comment:
Should we set these through constants (i.e. either in const but otherwise defined within the file itself?

Reply:
What do you mean? Write it like: `SENSORS = device["sensors"].
Or do you mean do this: SETPOINT_LOW = "setpoint_low", etc.?

Reply:
It's clear, implemented.

Reply:
👍